packages/create-wmr/tpl/public/index.js

```diff
@@ -27,6 +25,6 @@ export function App() {
 hydrate(<App />);
 
 export async function prerender(data) {
-	const { default: prerender } = await import('preact-iso/prerender');
-	return await prerender(<App {...data} />);
+	const { prerender: render } = await import('preact-iso');
```

Comment:
I think we can just have folks import `prerender` directly now, since it's already internally code-split:

```js
import { hydrate, prerender as ssr } from 'preact-iso';

export async function prerender(data) {
  return ssr(<App {...data} />);
}
```

(might also be a good time to alias it as `ssr`)

Reply:
done